tests/rule_based_profiler/parameter_builder/test_simple_date_format_string_parameter_builder.py

```diff
@@ -11,6 +11,11 @@
     ParameterContainer,
     get_parameter_value_by_fully_qualified_parameter_name,
 )
+from tests.integration.profiling.rule_based_profilers.conftest import (
+    alice_columnar_table_single_batch,
+    alice_columnar_table_single_batch_context,
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+)
```

Comment:
Wondering if we should duplicate these fixtures and move into unit test `conftest`.

Reply:
@NathanFarmer I see your point, but I'm just a bit uncomfortable with duplicating fixtures. Another option may be to move this to the `conftest.py` in the base `tests/` directory

@anthonyburdi @cdkini @alexsherstinsky would would we be able to get your thoughts on this point?

Reply:
@Shinnnyshinshin After our internal discussions I think we have agreed to move these into `tests/test_fixtures` and that should avoid the imports.